package.json

```diff
@@ -83,10 +66,33 @@
     "sass": "^1.26.12",
     "striptags": "^3.1.1",
     "styled-components": "^5.2.0",
-    "stylelint": "^13.8.0",
-    "stylelint-config-recess-order": "^2.3.0",
-    "stylelint-scss": "^3.18.0",
     "swr": "^0.5.6",
     "use-resize-observer": "^6.1.0"
+  },
+  "devDependencies": {
```

Comment:
relocated dev dependencies, consulted package pages to confirm if they are intended for dev use only